.github/dependabot.yaml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/images"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  versioning-strategy: lockfile-only
```

Comment:
I think the directory need to be /images/hub, and just lowered open-pull-request-limit to a month worth of PRs not that I think this will matter.

```suggestion
# Automatically update python packages for the jupyterhub/k8s-hub image
- package-ecosystem: pip
  directory: "/images/hub"
  schedule:
    interval: weekly
  open-pull-requests-limit: 4
  versioning-strategy: lockfile-only
```